commons/src/connections/availability.ts

```diff
@@ -0,0 +1,30 @@
+import {
```

Comment:
I was about to comment something like "we should consider having these connections named by function, not by component" but then I realized that both the component name (availability) and the function (fetching one's availability) are the same

so.... nice!